Init PR -> basic mod commands

FloareDor opened the pull request:
### v- Added basic mod commands -v
**

- Ban
- Unban
- Mute
- Unmute
- Kick

**

FloareDor committed:
Initial commit -> ban, unban, mute, unmute and kick commands

FloareDor committed:
Delete tempCodeRunnerFile.py

AnantaSrikar reviewed (changes requested):
Few issues that have to be addressed:

1. Bot isn't informing us if there are missing arguments.
2. The bot is not replying to non-admin users, which is fine, but there are un-handled exceptions.
3. You can delete the `admin_test` command, it was there for the template.
4. Remove unused imports in `cogs/admin/mod.py`.

FloareDor committed:
Updated to several edge cases

FloareDor committed:
Merge branch 'master' of https://github.com/FloareDor/Hawkeye

FloareDor committed:
Updated the bot to handle several edge cases

FloareDor commented:
#### Updated the bot to handle several edge cases
- The Bot now informs the user if there are any missing parameters and leads them to the `help` command.
- Removed the `admin_test` command.
- The Bot now also informs the user if an invalid command has been input.
- Removed unnecessary imports in `cogs/admin/mod.py`.
- Bot also notifies us if the mentioned user is not in the server.
- The help command now shows us how to type the commands as well.

AnantaSrikar reviewed (commented):
Good job! Everything seems to be working :+1: . Merging.
